### ISSUE TITLE
Build fails with "Cannot read property 'opId' of undefined"

### ISSUE THREAD

**Issue by wickedest** [swagger-api/swagger-ui#6210]
### Q&A (please complete the following information)
 - OS: ubuntu
 - Browser: chrome
 - Version: 83.0.4103.97 
 - Method of installation: npm
 - Swagger-UI version: 3.26.2
 - Swagger/OpenAPI version: Swagger 2.0, OpenAPI 3.0

### Describe the bug you're encountering
Our UI builds suddenly stopped working today, failing at the start of karma tests with the error:
```
HeadlessChrome 83.0.4103 (Linux 0.0.0) ERROR
  Uncaught TypeError: Cannot read property 'opId' of undefined
  at webpack:///home/jamie/piper/api-builder/node_modules/swagger-ui/dist/swagger-ui.js:8:0 <- test/loadtests.js:269997
```

I noticed that swagger-ui hadn't changed in a few days, however, swagger-client changed only 7 hours ago.  I also noticed this change: https://github.com/swagger-api/swagger-ui/commit/e3b3bf03d46f7288371cc505a73f7a49b75da1db#diff-c85b6c9e099af384a182e638a811c734R4 from July 6th.

Maybe it is possible that the `opId` change to swagger-client was a breaking change?

**Comment by AustinMutschler** [comment]
This has also happened to me last week. I am on v3.25.0 and this started happening out of no where.

**Comment by prashantsingh143** [comment]
Our build is also failing:-
Uncaught TypeError: Cannot read property 'opId' of undefined
    at Module.eval (swagger-ui.js?9474:8)
    at n (swagger-ui.js?9474:1)
    at Object.eval (swagger-ui.js?9474:1)
    at n (swagger-ui.js?9474:1)
    at eval (swagger-ui.js?9474:1)
    at eval (swagger-ui.js?9474:1)
    at eval (swagger-ui.js?9474:1)
    at eval (swagger-ui.js?9474:1)
    at Object../node_modules/swagger-ui/dist/swagger-ui.js (main.4d9867a7746ae1188ac6.js:91894)
    at __webpack_require__ (main.4d9867a7746ae1188ac6.js:790)

**Comment by hahanein** [comment]
We added `"swagger-client": "3.10.4"` to dependencies as a work-around.

**Comment by bahull** [comment]
Same error encountered today when installing codebase on new system

- OS: MacOS Catalina 10.15.2
- Browser: Chrome
  - Version: 83.0.4103.97
- Browser: Safari
  - Version: 13.0.4 (15608.4.9.1.3)
- Method of installation: npm
- Swagger-UI version: 3.22.2

*** Error:
```
Uncaught TypeError: Cannot read property 'opId' of undefined
    at Object.<anonymous> (swagger-ui.js:8)
    at __webpack_require__ (swagger-ui.js:1)
    at Object.<anonymous> (swagger-ui.js:8)
    at __webpack_require__ (swagger-ui.js:1)
    at Object.<anonymous> (swagger-ui.js:8)
    at __webpack_require__ (swagger-ui.js:1)
    at Object.<anonymous> (swagger-ui.js:1)
    at __webpack_require__ (swagger-ui.js:1)
    at Object.<anonymous> (swagger-ui.js:1)
    at __webpack_require__ (swagger-ui.js:1)
```

It appears that when swagger-ui tries to set
```
var opId = _swaggerClient.helpers.opId;
```

_swaggerClient.helpers is undefined.

**Comment by char0n** [comment]
Hello everybody, thanks for letting us know.

Along with already mentioned change in swagger https://github.com/swagger-api/swagger-ui/commit/e3b3bf03d46f7288371cc505a73f7a49b75da1db#diff-c85b6c9e099af384a182e638a811c734R4 we released [swagger-client](https://github.com/swagger-api/swagger-js) with [changes to our build artifacts](https://github.com/swagger-api/swagger-js/issues/1542). We're trying to figure out what is wrong at this point as we made sure new build artifacts are compatible with the old ones.

**Comment by char0n** [comment]
I could reproduce on simple Create React App setup. The problem is in this line in older version of swagger-ui:

https://github.com/swagger-api/swagger-ui/blob/3f06bb50fb31951f83dd6b78fa20aee77b6169f7/src/core/containers/OperationContainer.jsx#L4

`swagger-client` never had such an export and this only worked because of the fact that previous build artifact was exported as commonJS which has object-like semantics and support destructuring the same way as basic pojo.


Working on the fix.

**Comment by char0n** [comment]
`swagger-client` fixing PR addressing this issue provided here: https://github.com/swagger-api/swagger-js/pull/1631

**Comment by char0n** [comment]
We're just released [swagger-client@3.10.11](https://www.npmjs.com/package/swagger-client/v/3.10.11) that rectifies the issue. Please rerun your build and the problem should autoresolve. If you already have swagger-client@3.10.10 embeded in your `package-lock.json` or `yarn.lock` files please uninstall `swagger-ui` and then install it again.

Again, thanks for reporting this guys!
